docker/main/ngen/Dockerfile

```diff
@@ -65,9 +65,11 @@ ARG BOOST_VERSION=1.82.0
 #mpich 3.2 doesn't work well gfortran 11 it seems, an alignment error crops up, but 3.3.2 seems to work...
 ARG MPICH_VERSION="3.3.2"
 ARG MIN_PYTHON="3.8.0"
-ARG MIN_NUMPY="1.18.0"
 ARG BLOSC2_VERSION
 
+# Necessry to control specific numpy version after release of 2.0 and prior to fully support by all NextGen ecosystem
+ARG NUMPY_CONSTRAINT
+
```

Comment:
This is an abuse of the args parameter. Do not use this to inject python versions. Stick to setup.py and requirements.txt.